### PR TITLE
chore: rename prebuilt config classes and targets for clarity

### PR DESCRIPTION
**Pull request by anitarua**
Addresses #81 except for the docstrings; I'll come back to that with the docstrings ticket

### CODE REVIEW

File: Sources/Momento/config/CacheConfigurations.swift [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
                 )
             )
         }
-
-        public static func v1()  -> CacheClientConfiguration {
-            return latest()
+    }
+    
+    public enum macOS {
+        public static func latest() -> CacheClientConfiguration {
```
Comment (pgautier404) [inline Sources/Momento/config/CacheConfigurations.swift:15]: Can you go ahead and add a `default()` version for all of these these that just calls `latest()` please?

Comment (anitarua) [reply Sources/Momento/config/CacheConfigurations.swift:15]: hmm it gives me an error when trying to name it `default()`: `Keyword 'default' cannot be used as an identifier here`

Comment (anitarua) [reply Sources/Momento/config/CacheConfigurations.swift:15]: does `defaultConfig()` work as a name?

Comment (pgautier404) [reply Sources/Momento/config/CacheConfigurations.swift:23]: I think I'm good with that. @cprice404 what do you think about this naming?

Comment (pgautier404) [reply Sources/Momento/config/CacheConfigurations.swift:27]: Let's go with that for now. No need to block the PR on this. Please reach out to @cprice404 though at the next bug bash to verify he's ok with this naming.

**Review by pgautier404 — APPROVED** [review]

**Pull request merged**